inventory/management/misc/batch_transfers.rst

```diff
@@ -0,0 +1,126 @@
+=======================
+Process Batch Transfers
+=======================
+
+Batch picking allows a single picker to handle a batch of orders,
+reducing the number of times he must visit the same location. In Odoo,
+it means you will regroup several transfers into the same batch
+transfer, then processing it via its form view or the barcode
+application.
+
+Create a Batch Transfer
+=======================
+
+To activate the batch picking option, go to :menuselection:`Inventory
+--> Configuration --> Configuration` and enable *Batch Pickings*.
+
+.. image:: media/batch_transfers_01.png
+    :align: center
+    :alt: View of the inventory settings. Process to enable the batch pickings in the Odoo Inventory app
+
+Then, go to :menuselection:`Inventory --> Operations --> Batch
+Transfers` and hit the create button.
+
+.. image:: media/batch_transfers_02.png
+    :align: center
+    :alt: View of the new menu, Batch Transfers, under operations
+
+Now, determine the batch transfer responsible and the type of transfers
+you want to include in the batch. To do so, click on *Add a line*.
+
+.. image:: media/batch_transfers_03.png
+    :align: center
+    :alt: View of a Batch Transfers form
+
+For example, use the filter feature to only have the *Pick* step
+before shipping the products to customers. After that, select the
+different transfers which should be included in the batch.
+
+.. image:: media/batch_transfers_04.png
+    :align: center
+    :alt: View of the list of transfers to choose for a single batch transfer and how to add them
+          to the batch transfer
+
+To see the products to pick for the different transfers, click on
+confirm. The document also shows the locations they have been reserved
+from if *Multi-locations* has been activated.
+
+.. image:: media/batch_transfers_05.png
+    :align: center
+    :alt: View of a batch transfer list. Products to pick with their source and target locations
+
+Create a Batch Transfer from the Transfers List View
+====================================================
+
+From the *Transfers List View*, select transfers that should be
+included in the Batch. Then, hit *Add to batch* from the *Action*
+list.
+
+.. image:: media/batch_transfers_06.png
+    :align: center
+    :alt: View of the process to add transfers to a batch transfer from the transfers list view
+
+Next, determine if you want to add the transfers to an existing draft
+batch transfer or create a new one.
+
+.. image:: media/batch_transfers_07.png
+    :align: center
+    :alt: How to add a responsible to a batch transfer so it can be confirmed
+
+Process a Batch Transfer
+========================
+
+While gathering the products, you can edit the batch transfer and update
+the quantity done for each product. Once everything has been picked, hit
+validate so the different transfers contained in the batch are validated
+too.
```

Comment:
```suggestion
While gathering the products, you can edit the batch transfer and update
the *Quantity Done* for each product. Once everything has been picked, select
*Validate* so the different transfers contained in the batch are validated
too.
```

Suggestion just because are the exact names on the form, and I noticed you do it sometimes.
(+ it is something I do so it always calls my attention - If you want me to stop, let me know, please hehe)

Reply:
I don't mind if you keep suggesting that :) 
I just think people are smart enough to understand that "Validate" is a button, so I don't add * and uppercase for that one x)